rpc/wrtc_client.go

```diff
@@ -152,6 +155,18 @@ func dialWebRTC(
 		}
 	}()
 
+	var statsMu sync.Mutex
+	var callUpdates int
+	var maxCallUpdateDuration, totalCallUpdateDuration time.Duration
```

Comment:
minor

```suggestion
	var (
		statsMu sync.Mutex
		callUpdates int
		maxCallUpdateDuration, totalCallUpdateDuration time.Duration
	)
```

Reply:
Why isn't the former preferable? This looks like more lines without really affecting the readability.

Reply:
For me it helps to highlight the variables as a group since they are all related to stats, but I definitely don't feel strongly about this. Totally cool leaving as is.

Reply:
Grouped!